components/PillSwitch.tsx

```diff
@@ -37,71 +39,45 @@ const Slider = styled.div<{ isActive: boolean }>`
   font-size: 14px;
   font-weight: bold;
   z-index: 1;
+  pointer-events: none;
 
   @media only screen and (${devices.tablet}) {
     height: 48px; /* height of the slider */
   }
 `
 
-const SwitchInput = styled.input`
-  opacity: 0;
-  width: 0;
-  height: 0;
-`
-
-const TextLeft = styled.span`
+const DataGroupLink = styled(Link)`
   position: absolute;
   width: 50%;
-  left: 0px;
-  text-align: center;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
   color: ${({ theme }) => theme.offWhite};
-  pointer-events: none; /* ignore pointer events so label still triggers input */
-  z-index: 10;
-`
+  z-index: 20;
+  text-decoration: none;
 
-const TextRight = styled.span`
-  position: absolute;
-  width: 50%;
-  right: 0px;
-  text-align: center;
-  color: ${({ theme }) => theme.offWhite};
-  pointer-events: none;
-  z-index: 10;
+  &:first-of-type {
+    left: 0px;
+  }
+
+  &:last-of-type {
+    right: 0px;
+  }
 `
 
 type PillSwitchProps = {
-  onToggle: (isActive: boolean) => void
+  isActive: boolean
 }
 
-function PillSwitch({ onToggle }: PillSwitchProps) {
-  const [isActive, setIsActive] = useState(false)
-
-  const handleToggle = () => {
-    const newIsActive = !isActive
-    setIsActive(newIsActive)
-    onToggle(newIsActive)
-  }
-
-  const handleKeyDown = (event: React.KeyboardEvent) => {
```

Comment:
Since the PillSwitch uses regular links, they are keyboard accessible by default. Try tabbing to focus and then pressing enter. When I test in Firefox it works just like before.